app/subscriber/src/features/search-page/SearchPage.tsx

```diff
@@ -97,7 +97,7 @@ export const SearchPage: React.FC<ISearchType> = ({ showAdvanced }) => {
       });
       setCurrDateResults(currDateResults);
       setPrevDateResults(prevDateResults);
-      setTotalResults(currDateResults.length + prevDateResults.length);
+      setTotalResults(currDateResults.length);
```

Comment:
This would lead to incorrect total results as it could be including content not shown on screen